### DISCUSSION TITLE
Multi-Tenancy with custom angular app and abp aspnet core api

### DISCUSSION THREAD

**Discussion by coderhawk** [abpframework/abp#16936] [General] [answered]
Hi,

I have one abp io aspnet core API with multi-tenant enabled; and one custom angular app which is not using any of the abp io components. 

I would like to know how to pass the tenant id from angular to api ?

Please share some example code if possible.

**Comment by maliming** [comment, marked as answer]
hi

https://docs.abp.io/en/abp/latest/Multi-Tenancy#tenant-resolvers
